api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
 # Create your models here.
+
+class Task (models.Model):
+
+	STATUS_CHOICES = (
+		('UNASSIGNED', 'Unassigned'),
+		('WORKINGON', 'Working On'),
+		('COMPLETED', 'Completed'),
+		)
```

Comment:
Yes @goranche : I like it!
Maybe could be like this: 
UNASSIGNED (by default) + assigned_id blank = unassigned
assigned_id not blank = assigned --> status_id changes automatically to 'ASSIGNED'
'WORKINGON' - when the the task is under work the assigned user change manually to 'WORKINGTON'
'COMPLETED' is when task is completed

So I would like to add the 'ASSIGNED' value to the choices.